From 7c9ae7157ddbca6a1d9ac6c4f7e7395f1541a898 Mon Sep 17 00:00:00 2001
From: Isaac Davis <isaac.davis@joyent.com>
Date: Thu, 16 May 2019 01:14:36 +0000
Subject: [PATCH] TRITON-1670 sdcadm AddAllowTransferProcedure uses incorrect
 parameter name when listing service Reviewed by: Trent Mick
 <trentm@gmail.com> Approved by: Trent Mick <trentm@gmail.com>

---
 CHANGES.md                           | 4 ++++
 lib/procedures/add-allow-transfer.js | 4 ++--
 package.json                         | 2 +-
 3 files changed, 7 insertions(+), 3 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index 2bbb1b6..a9e9d7a 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,10 @@
 
 # sdcadm Changelog
 
+## 1.27.1
+
+- TRITON-1670 sdcadm AddAllowTransferProcedure uses incorrect parameter name when listing service
+
 ## 1.27.0
 
 - TRITON-1625 Have `sdcadm post-setup prometheus` add prometheus zone IP to CNS allow\_transfer list
diff --git a/lib/procedures/add-allow-transfer.js b/lib/procedures/add-allow-transfer.js
index 79a24af..8df90a0 100644
--- a/lib/procedures/add-allow-transfer.js
+++ b/lib/procedures/add-allow-transfer.js
@@ -246,7 +246,7 @@ AddAllowTransferProcedure.prototype.doVolatileOperations =
             function getCnsSvc(_, next) {
                 sapi.listServices({
                     name: 'cns',
-                    application: sdcadm.sdcApp.uuid
+                    application_uuid: sdcadm.sdcApp.uuid
                 }, function gotCnsSvc(err, svcs) {
                     if (err) {
                         next(new errors.SDCClientError(err, 'sapi'));
@@ -261,7 +261,7 @@ AddAllowTransferProcedure.prototype.doVolatileOperations =
             function getSvcUuid(ctx, next) {
                 sapi.listServices({
                     name: svcName,
-                    application: sdcadm.sdcApp.uuid
+                    application_uuid: sdcadm.sdcApp.uuid
                 }, function gotSvcs(err, svcs) {
                     if (err) {
                         next(new errors.SDCClientError(err, 'sapi'));
diff --git a/package.json b/package.json
index 13cf54c..18a5923 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdcadm",
   "description": "Administer a Triton Data Center",
-  "version": "1.27.0",
+  "version": "1.27.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-- 
2.21.0

